### ISSUE TITLE
Diamond Windows Installation Question

### ISSUE THREAD

**Issue by CW1211** [bbuchfink/diamond#779]
Hello, I am trying to download Diamond to a Windows computer. I downloaded Visual C++ Redistributable from Microsoft (https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170). I also downloaded the binary executable file from here to my desktop (https://github.com/bbuchfink/diamond/releases). When I just open diamond.exe, nothing happens. I saw in a previous post that you need to run it from the command prompt. I have tried doing so from the Windows Command Prompt but I get the following error message: "'diamond.exe' is not recognized as an internal or external command,
operable program or batch file." 

Can someone help me figure out the issue here? Do I need to download a different version of Visual CC+ Redistributable? Thanks!

**Comment by bbuchfink** [comment]
That error message means that the `diamond.exe` was not found. Make sure that it is in the current working directory.

**Comment by CW1211** [comment]
Hi, I moved and extracted the diamond.exe file to my current working directory (which I checked by using the cd command in the Windows Command Prompt). It still will not run. So could the Visual C++ Redistributable I am using be the issue? I could not download the 2015 version listed on the Installation guide so I used the Visual Studio for 2015, 2017, 2019, and 2022 package from Microsoft (https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170).
